src/caldav/src/FreeBusyOrgScopePlugin.php

```
@@ -54,6 +63,72 @@ public function beforePost(RequestInterface $request)
+
+        $sharingLevel = $request->getHeader('X-LS-Org-Sharing-Level');
+
+        // sharing_level=none: block all freebusy (even same-org)
+        if ($sharingLevel === 'none') {
+            throw new DAV\Exception\Forbidden(
+                'Free/busy queries are not allowed when sharing is disabled'
+            );
+        }
+
+        // Cross-org: always blocked
+        $requesterOrgId = $request->getHeader('X-LS-Org-Id');
```

PHPStan (2.1.46)

[error] Call to method getHeader() on an unknown class Sabre\HTTP\RequestInterface.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)